posthog/models/integration.py

```diff
@@ -632,7 +632,7 @@ def dfs(account_id, accounts=None, parent_id=None) -> list[dict]:
             )
 
             if response.status_code != 200:
-                return []
+                return accounts
```

Comment:
I don't follow tbh - why if the status is not 200 do we return the accounts? I feel like this needs comments and tests